docs/UserGuide.md

```diff
@@ -133,6 +135,15 @@ Examples:
 * `findp alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
```

Comment:
Should we cut away this image since it is the incorrect example?

Reply:
Will remove this!